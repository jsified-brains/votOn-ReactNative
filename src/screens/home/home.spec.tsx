import React from 'react';
import HomeScreen from './home';
import renderer from 'react-test-renderer';

describe('Home Component.', () => {
  it('matches expected snapshot', () => {
    const header = renderer.create(
      <HomeScreen />
    ).toJSON;
    expect(header).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const rendered = renderer.create(<HomeScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });
});
