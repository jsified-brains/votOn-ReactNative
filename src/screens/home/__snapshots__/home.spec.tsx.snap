// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component. matches expected snapshot 1`] = `[Function]`;
